Block/Adminhtml/Akeneo/Edit.php

```diff
@@ -50,7 +50,7 @@ protected function _construct(): void
     /**
      * Retrieve text for header element depending on loaded post
      */
-    public function getHeaderText(): Phrase
+    public function getHeaderText(): Phrase|string
```

Comment:
Uit nieuwsgierigheid, is de string type hier wel nodig? Ik zie in de code alleen Phrases staan, misschien vergeten weg te halen?

Reply:
Ik kreeg hier een melding voor van PHPStan, vandaar dat `string` is toegevoegd. Wellicht dat er ook een string uit kan komen?